modules/osdk-java-controller-memcached-deployment.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/java/osdk-java-tutorial.adoc
+
+:_content-type: CONCEPT
+[id="osdk-java-controller-memcached-deployment_{context}"]
+=  Defining `createMemcachedDeployment`
```

Comment:
Only procedures and assemblies start with [gerunds](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings).